api/api.js

```diff
@@ -0,0 +1,18 @@
+import fetch from 'isomorphic-fetch';
+
+
+const api = {
+    urlApi: 'https://pereiratechtalks.com',
+    PTTpost: {
+        async posts(){
+            const queryParams = '?limit=9&filter=author%3Apereirajs&client_id=ghost-frontend&client_secret=7709d8d5ac20'
```

Comment:
Just to make sure: Is it safe to expose the `client_secret` token publicly?

Reply:
Hi @jonalvarezz  it seems like this is okay, We were talking about this in the telegram group...or at least that's what @pin3da just told me, you're going to get all the ghost post from ptt

Reply:
Great, thanks